test/images/dnsutils/Dockerfile_windows

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
In few places I see that `dnsmasq` command has been called part of pod, not sure that case is handled here! - https://github.com/kubernetes/kubernetes/blob/2a05e1b76f9f0fa52a31701ba0074d8d431da2ec/test/e2e/network/dns_common.go#L292

Reply:
True, and it is not handled. I haven't found a Windows equivalent for that yet, but this will be sufficient for most DNS tests to pass.